spm_funcs.py

```diff
@@ -54,8 +54,16 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    img = nib.load(fname) # Load image
+    data = img.get_fdata() # Get image data
+
+    spm_values = np.asarray([]) # create empty array for spm values per volume
```

Comment:
You can delete this line - you aren't using it.

Reply:
of course, thank you. I started a for loop then converted to a list comprehension and forgot to del. Fixing...